stories/core/Buttons/ButtonGroup.stories.tsx

```diff
@@ -119,3 +121,27 @@ Overflow.parameters = {
     ignoreElements: ['small'],
   } as CreeveyStoryParams,
 };
+
+export const InputButtonCombo: Story<ButtonGroupProps> = (args) => {
```

Comment:
So you said you fixed it?
![input-button-combo](https://user-images.githubusercontent.com/36186912/146794071-4a5d7b20-5cab-41f3-bad5-2a34db94a5bc.gif)

Reply:
Did you run `yarn` and restart storybook?

Reply:
Yup and ran on new folder freshly.

Reply:
![image](https://user-images.githubusercontent.com/36186912/146794934-28fdcab3-9038-4b41-a39d-379096d2818a.png)
It seems that there is your latest changes.

Reply:
wow, now it's inconsistent. It worked for me, and then I refreshed and it broke, and then I refreshed again and it works.

This is a bug in storybook and I am not sure what to do.

Reply:
Any idea which part of storybook is causing this?

Reply:
Its working fine for me

Reply:
> Its working fine for me

Does it consistently work fine? Try refreshing a few times and testing each time, and also try incognito mode

Reply:
Oh yeah randomly stops working 😕

Reply:
Weirdly, it works fine in Firefox as well as Edge and Brave (both of which use Chromium). So maybe it's just a Chrome bug with positioned iframes.

Reply:
Was not able to reproduce this in the latest versions of Safari, Firefox, or Brave.  I am able to reproduce on the latest version of Chrome though (Version 96.0.4664.110 (Official Build) (x86_64)).

Reply:
This PR should fix the issue: https://github.com/iTwin/iTwinUI/pull/422

Reply:
Fixed, please check now. @bentleyvk @saaaaaally

Reply:
Yes, its working now for me.